rule-types/github/sca_dependabot.yaml

```
@@ -0,0 +1,159 @@
+---
+version: v1
+release_phase: beta
+type: rule-type
+name: sca_dependabot
+display_name: "SCA automation: Dependabot"
+short_failure_message: "SCA: Dependabot is not configured"
```

The Semgrep rule has:

```yaml
display_name: Enable Semgrep SAST
short_failure_message: SAST is not configured
```

So, should this be:

```suggestion
display_name: "Enable Dependabot SCA"
short_failure_message: "Dependabot SCA is not configured"
```

and the Semgrep one change to:

```suggestion
display_name: "Enable Semgrep SAST"
short_failure_message: "Semgrep SAST is not configured"
```

---

I'm actually going to split the difference, and have the rule name be "Enable $product $TLA", but the short failure message be "$TLA: $product is not configured".  The idea is that the short failure message may be displayed in a summary view on a webpage or in a report, and starting with the short abbreviation of the functionality will enable easier sorting of results into categories.